pandas-stubs/core/series.pyi

```diff
@@ -851,7 +851,7 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     @overload
     def apply(
         self,
-        func: Callable[..., Scalar | Sequence | set | Mapping | None],
+        func: Callable[..., Scalar | Sequence | set | Mapping | NAType | None],
```

Comment:
I think this change came up because you haven't merged with `main`.  Can you make sure you have done that?